src/main/java/com/harragan/battleshipsboot/BattleshipsbootApplication.java

```diff
@@ -1,8 +1,10 @@
-package uk.gov.ukho.battleshipsboot;
+package com.harragan.battleshipsboot;
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
 
+@EnableJpaRepositories
```

Comment:
If you have spring-boot-starter-data-jpa then you shouldn't need this annotation